src/Adapters/Auth/phantauth.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Helper functions for authenticating with PhantAuth.
+ *
+ * PhantAuth was designed to simplify testing for applications using OpenID Connect
+ * authentication by making use of random generated users.
+ *
+ * To learn more, please go to: https://wwww.phantauth.net
```

Comment:
https://www.phantauth.net, you have an extra w

Reply:
thanx, fixed